packages/preferences-controller/src/PreferencesController.ts

```diff
@@ -114,6 +114,18 @@ export type PreferencesState = {
    * Controls whether Multi rpc modal is displayed or not
    */
   useMultiRpcMigration: boolean;
+  /**
+   * Controls whether to use the safe chains list validation
+   */
+  useSafeChainsListValidation: boolean;
+  /**
+   * Controls which order tokens are sorted in
+   */
+  tokenSortConfig: Record<string, string>;
```

Comment:
This type tells consumers that they can't make any assumptions about what's in this object. Is this true though? For instance, it seems that `order` within this object can either be `asc` or `dsc`, but I can't think of a reason why we'd allow anything else.

Should this be more specific?

Reply:
I just added TokenSortConfig type